components/base/kinematicbase/execution.go

```diff
@@ -331,9 +331,9 @@ func (ptgk *ptgBaseKinematics) courseCorrect(
 		"allowable diff ", allowableDiff, " diff now ", poseDiff.Point().Norm(), " angle diff ", poseDiff.Orientation().AxisAngles().Theta,
 	)
 
+	ptgk.logger.Debug("expected to be at ", spatialmath.PoseToProtobuf(expectedPose))
+	ptgk.logger.Debug("Localizer says at ", spatialmath.PoseToProtobuf(actualPose.Pose()))
```

Comment:
I don't think we want this log change to be permanent, it would be super chatty.

Reply:
sounds good, I will revert.